go.mod

```diff
@@ -1,47 +1,42 @@
 module github.com/fluxcd/helm-operator
 
-go 1.12
+go 1.13
+
+// TODO(carnott-snap): this replace is a noop, but required till fluxcd/flux#2590 is released
+replace github.com/docker/distribution => github.com/docker/distribution v2.7.1+incompatible
+
+// TODO(carnot-snap): force downgrade because of a trasitive upgrade
+//
+// github.com/fluxcd/helm-operator
+// +-> github.com/fluxcd/flux@v1.15.0
+//     +-> github.com/fluxcd/helm-operator@v1.0.0-rc1
+//         +-> github.com/weaveworks/flux@v0.0.0-20190729133003-c78ccd3706b5
+//             +-> k8s.io/code-generator@v0.0.0-20190511023357-639c964206c2
+//
+// This replace should be removed once fluxcd/flux is released.
+replace k8s.io/code-generator => k8s.io/code-generator v0.0.0-20190311093542-50b561225d70
```

Comment:
removal of this dependency cycle will require synchronised commits in both `helm-operator` and `flux`:
1. remove all `require` and `replace` blocks from helm-operator: carnott-snap/helm-operator@100d4a6e2252dc3a988c1a3aec2dcbdccc560763
2. pin `flux` to pseudo version of `helm-operator`: carnott-snap/flux@65da8ef3744eaa2b70445f2fcaaee95b8d033ea6
3. remove unnecessary `replace` blocks:
    - carnott-snap/helm-operator@d8142acbe99b9d93a4a4ea8e5ff1e06118427903
    - carnott-snap/flux@a8f13dff40162434e67c324eebbb6905033c52d7